src/Aspire.Dashboard/Components/Controls/PlotlyChart.razor.cs

```diff
@@ -173,7 +173,7 @@ private sealed class Trace
 
     private string FormatTooltip(string name, double yValue, DateTime xValue)
     {
-        return $"<b>{InstrumentViewModel.Instrument?.Name}</b><br />{name}: {yValue.ToString("##,0.######", CultureInfo.InvariantCulture)}<br />Time: {xValue.ToString("h:mm:ss tt", CultureInfo.InvariantCulture)}";
+        return $"<b>{InstrumentViewModel.Instrument?.Name}</b><br />{name}: {yValue.ToString("##,0.######", CultureInfo.CurrentCulture)}<br />Time: {xValue.ToString("h:mm:ss tt", CultureInfo.CurrentCulture)}";
```

Comment:
When using a custom format string, does the culture even matter?

To that point, should we avoid hard coding a format string here.

```suggestion
        return $"<b>{InstrumentViewModel.Instrument?.Name}</b><br />{name}: {yValue.ToString("N6")}<br />Time: {xValue.ToString("T")}";
```

Reply:
N6 forces decimal points, even if they're all zero. We don't want that.

The date should be changed to a help method though.

Reply:
Then the pattern should be built up using constants that respect the user's/culture's decimal and thousand separators.

- `NumberFormatInfo.CurrentInfo.NumberDecimalSeparator`
- `NumberFormatInfo.CurrentInfo.NumberGroupSeparator`

Reply:
From https://en.wikipedia.org/wiki/Decimal_separator

![image](https://github.com/dotnet/aspire/assets/350947/9aedd209-f656-477b-819a-46f5e8cd6587)

Reply:
dot and comma are automatically replaced as needed:

https://learn.microsoft.com/en-us/dotnet/standard/base-types/custom-numeric-format-strings

Reply:
TIL